controllers/utils/s3.go

```diff
@@ -10,6 +10,22 @@ import (
 const (
 	RamenHubNamespace  = "openshift-dr-system"
 	BucketGenerateName = "odrbucket"
+
+	// rook
+	RookOrigin = "rook"
```

Comment:
RookOrigin is not exactly  S3. 
  Ideally, we should create OriginMap instead of keeping separate variables for each Origin: RookOrigin, S3Origin.

Reply:
removed the consts and created a map for the origin